core-bundle/src/Resources/contao/templates/backend/be_main.html5

```diff
@@ -40,6 +40,7 @@
       $objCombiner->add('bundles/contaocore/limit-height.min.js');
       $objCombiner->add('bundles/contaocore/tips.min.js');
       $objCombiner->add('system/themes/'.$this->theme.'/hover.min.js');
+      $objCombiner->add('bundles/contaocore/sectionwizard.min.js');
```

Comment:
Did you add this after the `hover.min.js` script on purpose? If not, we should probably move it up.

Reply:
No, I just added it last, since it's a new entry.